docs/getting-started/first-release.md

```diff
@@ -0,0 +1,199 @@
+
+In a blue-green rollout, a percentage of requests are directed to a candidate version of the model. This percentage can be changed over time.
+
+![Blue-green rollout](../tutorials/images/blue-green.png)
+
+In this tutorial, we use the Istio service mesh to distribute inference requests between different versions of a model.
+
+???+ warning "Before you begin"
+    1. Ensure that you have a Kubernetes cluster and the [`kubectl`](https://kubernetes.io/docs/reference/kubectl/) and [`helm`](https://helm.sh/) CLIs. You can create a local Kubernetes cluster using tools like [Kind](https://kind.sigs.k8s.io/) or [Minikube](https://minikube.sigs.k8s.io/docs/).
+    2. Install [Istio](https://istio.io). It suffices to install the [demo profile](https://istio.io/latest/docs/setup/getting-started/), for example by using: 
+    ```shell
+    istioctl install --set profile=demo -y
+    ```
+
+## Install the Iter8 controller
+
+--8<-- "docs/getting-started/install.md"
+
+```shell
+export IMG=kalantar/iter8:20231011-1506
+export CHARTS=/Users/kalantar/projects/go.workspace/src/github.com/iter8-tools/iter8/charts
+helm upgrade --install iter8 $CHARTS/controller \
+--set image=$IMG --set logLevel=trace \
+--set clusterScoped=true
+```
+
+## Deploy initial version
+
+Deploy the initial version of the application using the Iter8 `release` chart by identifying the environment into which it should be deployed, a list of the versions to be deployed (here just one), and the rollout strategy to be used:
+
+```shell
+cat <<EOF | helm upgrade --install httpbin $CHARTS/release -f -
+environment: deployment-istio
+application: 
+  versions:
+  - metadata:
+      labels:
+        app.kubernetes.io/version: v0
+    image: kennethreitz/httpbin
+  strategy: blue-green
+EOF
+```
+
+??? note "What happens?"
+    Because `environment` is set to `deployment-istio`, a `Deployment` and a `Service` object are created.
+        - The namespace `default` is inherited from the helm release namespace since it is not specified in the version or in `application.metadata`.
+        - The name `httpbin-0` is derived from the helm release name since it is not specified in the version or in `application.metadata`. The names is derived by appending the index of the version in the list of versions; `-0` in this case.
+        - Alternatively, a `deploymentSpecification` and/or a `serviceSpecification` could have been specified.
+
+    To support routing, a `Service` (of type `ExternalName`) named `default/httpbin` pointing at the Istio gateway, `istio-ingressgateway.istio-system`, is deployed. The name is the helm release name since it not specified in `application.metadata`. Further, an Iter8 [routemap](../user-guide/topics/routemap.md) is created. Finally, to support the blue-green rollout, a `ConfigMap` (`httpbin-0-weight-config`) is created to be used to manage the proportion of traffic sent to this version.
```

Comment:
```suggestion
    To support routing, a `Service` (of type `ExternalName`) named `default/httpbin` pointing at the Istio gateway, `istio-ingressgateway.istio-system`, is deployed. The name is the Helm release name since it not specified in `application.metadata`. Further, an Iter8 [routemap](../user-guide/topics/routemap.md) is created. Finally, to support the blue-green rollout, a `ConfigMap` named `httpbin-0-weight-config` is created to be used to manage the proportion of traffic sent to this version.
```